src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest/Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" />
```

Comment:
Are you sure that you need this package? That's the very big meta package that is only needed when doing msbuild evaluation or similar.

Reply:
Looks like it's not needed, at least it built fine on my machine